examples/training/classification/imagenet/basic_training.py

```diff
@@ -85,6 +85,30 @@
     "Keyword argument dictionary to pass to the constructor of the model being trained",
 )
 
+flags.DEFINE_string(
+    "learning_rate_schedule",
+    "ReduceLROnPlateau",
```

Comment:
Sure, updating! I went with the class name out of inertia haha
You mean, create `REDUCE_ON_PLATEAU = "ReduceOnPlateau"` (and one for cosine) to avoid writing the string out or?

Reply:
yeah that's what I was thinking. Helps make sure we avoid typos :)

Reply:
Done :)